### PR TITLE
Geoserver debian package

### PR DESCRIPTION
**Pull request by pmauduit**
As you can see, almost nothing changed. To make the review easier, I suggest you to open the 2nd commit directly.

To generate a debian package:
```
mvn clean install
mvn package deb:package -pl webapp/
```
(supposing you are in the geoserver/ directory)

### CODE REVIEW

File: geoserver/webapp/pom.xml [hunk 1]
```diff
@@ -762,5 +758,137 @@
...
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>net.sf.debian-maven</groupId>
+            <artifactId>debian-maven-plugin</artifactId>
+            <version>1.0.6</version>
+            <configuration>
+              <packageName>georchestra-geoserver</packageName>
+              <packageDescription>Debian package for the geOrchestra's GeoServer</packageDescription>
```
Comment (fvanderbiest) [inline geoserver/webapp/pom.xml]: geOrchestra GeoServer Debian package

File: geoserver/webapp/pom.xml [hunk 2]
```diff
@@ -762,5 +758,137 @@
...
+              <packageDependencies>
+                <packageDependency>debconf</packageDependency>
+              </packageDependencies>
+              <projectUrl>http://www.georchestra.org/</projectUrl>
+              <projectOrganization>geOrchestra</projectOrganization>
+              <maintainerName>Pierre Mauduit</maintainerName>
+              <maintainerEmail>pierre.mauduit@gmail.com</maintainerEmail>
```
Comment (fvanderbiest) [inline geoserver/webapp/pom.xml]: PSC

File: geoserver/webapp/pom.xml [hunk 3]
```diff
@@ -762,5 +758,137 @@
...
+      <id>rpmPackage</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-war-plugin</artifactId>
+            <version>2.1.1</version>
+            <configuration>
+              <classifier>generic</classifier>
+              <packagingExcludes>**/docs/**,**/openlayers/examples/**,**/openlayers/tests/**</packagingExcludes>
```
Comment (fvanderbiest) [inline geoserver/webapp/pom.xml]: useless

File: geoserver/webapp/src/deb/resources/etc/georchestra/geoserver/geoserver.properties [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# GeoServer has already its own datadir system, in addition this has to be
+# read-write by the user running the application server, which is unlikely the
+# case in /etc/
+#
+# Please configure the GeoServer application server with the following
+# environment variable:
+#
+# export GEOSERVER_DATA_DIR=/path/to/geoserver/datadir
+#
+# A minimal datadir for geOrchestra's needs can be found here:
```
Comment (fvanderbiest) [inline geoserver/webapp/src/deb/resources/etc/georchestra/geoserver/geoserver.properties]: A minimal datadir for geOrchestra is provided here:

**Comment by fvanderbiest** [comment]
LGTM otherwise

File: geoserver/webapp/pom.xml (hunk 1 above)
Comment (pmauduit) [inline geoserver/webapp/pom.xml]: As discussed with @landryb on IRC, let's keep "geOrchestra GeoServer" only (no need to mention "debian package")

File: geoserver/webapp/pom.xml (hunk 2 above)
Comment (pmauduit) [inline geoserver/webapp/pom.xml]: done

File: geoserver/webapp/pom.xml (hunk 3 above)
Comment (pmauduit) [inline geoserver/webapp/pom.xml]: done

**Pull request merged**